artiq/gateware/eem.py

```diff
@@ -222,13 +222,13 @@ def io_qspi(eem0, eem1, iostandard):
         return ios
 
     @classmethod
-    def add_std(cls, target, eem, eem_aux, ttl_out_cls, sync_gen_cls=None, iostandard=default_iostandard):
+    def add_std(cls, target, eem, eem_aux, ttl_out_cls, sync_gen_cls=None, iostandard=default_iostandard, dds=None):
```

Comment:
Why is ``None`` an appropriate default value?
Also I think all the other methods have ``iostandard`` last.

Reply:
done